embulk-output-redshift/src/main/java/org/embulk/output/RedshiftOutputPlugin.java

```diff
@@ -193,6 +201,6 @@ protected BatchInsert newBatchInsert(PluginTask task, Optional<MergeConfig> merg
         RedshiftPluginTask t = (RedshiftPluginTask) task;
         setAWSCredentialsBackwardCompatibility(t);
         return new RedshiftCopyBatchInsert(getConnector(task, true),
-                getAWSCredentialsProvider(t), t.getS3Bucket(), t.getS3KeyPrefix(), t.getIamUserName(), t.getDeleteS3TempFile());
+                getAWSCredentialsProvider(t), t.getS3Bucket(), t.getS3KeyPrefix(), t.getIamUserName(), t.getDeleteS3TempFile(), t.getCopyIamRoleName().or(""), t.getCopyAwsAccountId().or(""));
```

Comment:
`or("")` should be `orNull()` because the default value is defined as `null`.

Reply:
I didn't check orNull, It's better for this, I'll fix.